tests/test_scripts.py

```diff
@@ -1,22 +1,31 @@
 import os
 from entity_resolver.scripts import run
+import json
```

Comment:
Move `import json` before the last line `from entity_resolver.scripts import run`.